docs/synapse/adminguides/permissions.rstorm

```diff
@@ -0,0 +1,590 @@
+
+**Examples:**
+
+Revoke the rule that prevents user "ron" from setting tag descriptions:
+
+.. storm-cli:: auth.user.delrule ron "!node.prop.set.syn:tag:desc"
+
+
+Revoke the rule that allows "junior analysts" to apply tags in the ``cno.threat`` tag tree:
+
+.. storm-pre:: auth.role.add "junior analysts"
+.. storm-pre:: auth.role.addrule "junior analysts" node.tag.cno.threat
+
+.. storm-cli:: auth.role.delrule "junior analysts" node.tag.cno.threat
+
+
+.. _admin_authgate_perms:
+
+Auth Gate Permissions
+---------------------
+
+To assign permissions for an Auth Gate, you use the same Storm commands used to assign global permissions, but
+you must specify the Auth Gate's full identifier (iden) (using the ``--gate`` option) when adding or
+removing the rule.
+
+Obtain a Gate's Iden
+~~~~~~~~~~~~~~~~~~~~
+
+The Storm :ref:`storm-view` and :ref:`storm-layer` commands can be used to manage views and layers, respectively.
+In particular, the following commands are useful for displaying all views or layers (including their idens), or
+displaying a specific view or layer:
+
+- :ref:`storm-view-list`
+- :ref:`storm-view-get`
+- :ref:`storm-layer-list`
+- :ref:`storm-layer-get`
+
+**Example:**
+
+Display all views:
+
+.. storm-cli:: view.list
+
+Viewing a Gate's Permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``auth.gate.show`` command is used to display permissions information about a particular Auth Gate (e.g.,
+a view or layer). You can provide the specific iden for an Auth Gate, or use the syntax below to retrieve
+information for the **current** view or layer. (Viewing information for the "current layer" will return
+information for the top layer of the current view.)
+
+**Example:**
+
+Display information for the current view:
+
+.. storm-cli:: auth.gate.show $lib.view.get().iden
+
+Display informatiton for the current layer:
+
+.. storm-cli:: auth.gate.show $lib.layer.get().iden
+
+
+.. _admin_axon_perms:
+
+Axon Permissions
+================
+
+You must specifically grant users or roles permission to access (upload, check, or retrieve) files within a
+storage Axon.
+
+- See the :ref:`deploy_axon` section of the :ref:`deploymentguide` for information on Axon deployment.
+- See the :ref:`devops-svc-axon` section of the :ref:`devopsguide` for information on Axon configuration,
+  including a description of Axon permissions.
+
+
+.. _admin_optic_perms:
+
+Optic Permissions
+=================
+
+Commercial Synapse customers with the Optic UI may need to explicitly grant users or roles permission to some
+UI tools (such as Spotlight).
+
+- See the `Optic Deployment Guide`_ for information on Optic deployment.
+- See the `Optic DevOps Guide`_ for information on Optic permissions and other features.
+
+.. TIP::
+  
+  You do not need to explicitly grant permissions to Optic itself. If you are creating and managing Synapse
+  ("Cortex") users and roles via Optic, they have permission to access Optic by default. (New users will be
```

Comment:
``will be`` -> ``may be``. That is a deployment specific configuration.

Reply:
removed sentence